### PR TITLE
Use Prettier to Auto-Format the Code

### PR DESCRIPTION
**Pull request by GenieTim**
This is a very opinionated PR; while it includes the changes from #14, its main contribution is the set-up and initial run of the opinionated code formatter [Prettier](https://prettier.io/). While this is an arbitrary choice — as is Prettier's default configuration — I would highly suggest you offer some configuration with a corresponding tool in order to have consistent PRs and reduce merge-conflicts from people with different auto-formatters in their IDE. This here would be my suggestion on how this could be.

One more little thing that it includes: in index.html, there seems to be an unclosed `<div>` and in turn a closing `</article>` without the corresponding opening tag — this would be fixed with this PR, too.

Thanks anyway!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,193 +1,270 @@
...
+          <a
+            class="underline hover:no-underline"
+            href="https://github.com/rosswintle/can-i-php/blob/main/README.md"
+            >README.md</a
+          >
+          &mdash;
+          <a
+            class="underline hover:no-underline"
+            href="https://github.com/rosswintle/can-i-php"
+            >Github</a
```
Comment (tyrann0us) [inline index.html]:
```suggestion
            >GitHub</a
```

**Comment by rosswintle** [comment]
@GenieTim Thanks for this. Code standards are good! I appreciate the contribution.

Please don't take my lack of reply as me not being interested. I've just not used Prettier before and want to figure out what this is doing before I merge it.

**Comment by GenieTim** [comment]
@rosswintle I am sorry, it seems that checkbox does not exist as I accidentally seemed to have forked the repo as an organisation.

See https://github.com/isaacs/github/issues/1681 and https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847/8 and https://github.com/whatwg/html/pull/3881#issuecomment-668467097 as to why I think this is the case.

Would it help if I gave you maintainer access to my fork? Or shall I push and PR again from a private fork?

**Comment by rosswintle** [comment]
This will be fixed by #18 which has had the main branch merged in.

**Pull request closed without merging**